packages/fastify-vite/package.json

```diff
@@ -46,7 +46,7 @@
   },
   "version": "8.0.0-alpha.2",
   "optionalDependencies": {
-    "vite": "^5.4.8"
+    "vite": "^5.4.16"
```

Comment:
Can you use the catalog feature in pnpm-workspace.yaml instead of re-declaring the same version over and over?